common/http/src/main/java/io/helidon/common/http/CharMatcher.java

```diff
@@ -54,16 +54,21 @@ protected CharMatcher() {
     /**
      * Determines whether a character is ASCII, meaning that its code point is less than 128.
      *
-     * @since 19.0 (since 1.0 as constant {@code ASCII})
+     * @return this CharMatcher instance
```

Comment:
The javadoc is wrong. You cannot return "this instance" as this is a static method